### ISSUE TITLE
Package individually not working with pipenv modules

### ISSUE THREAD

**Issue by username-dot-com** [UnitedIncome/serverless-python-requirements#268]
I have a service where each function is a module using pipenv to manage dependencies for each module. The plugin doesn't generate requirements.txt using pipenv unless I have a Pipfile in the root of the service, even if I have them in the modules. However, when I do that, the plugin generates a requirements.txt for each module using the root pipfile instead of the pipfile in the module itself.

I think the issue might be in the function "pipfileToRequirements()" in pipenv.js. It always passes the service root to the spawnSync call.

Packaging seems to work if I manually generate the requirements.txt in each module prior to sls deploy/package. Though there is still another issue where serverless generates two zips per function. One zip is named the lambda function name specified in serverless.yml that follows include/exclude but otherwise would contain the entire service. The other is a zip named the lambda function after it is deployed which I believe is the one generated by this plugin. My understanding according to the docs is there should only be 2 zips?

I'm not sure if these two issues are related to each other nor if the second issue isn't really an issue and is just how it works currently, so I'd like some insight into this if possible.

I've created a reproduction repo for this here:
[https://github.com/username-dot-com/sls-python-test](https://github.com/username-dot-com/sls-python-test)

**Comment by dschep** [comment]
Good catch. That was probably over looked in the package individually implementation. cc @cgrimal

**Comment by FelschR** [comment]
It would be nice if this were documented somewhere.
Tried to setup the plugin with pipenv yesterday for quite a while until I found this issue.

**Comment by glumia** [comment]
This is still an issue and should be labelled/treated as a bug, it is not documented nor intuitive that `individually: true` doesn't work with Pipenv/Poetry modules.

**Comment by psvaiter** [comment]
Is it already fixed? I took a long time searching for a solution until I find this issue. I was suspecting that Pipfile was not supported in modules.

**Comment by pgrzesik** [comment]
It seems to not be fixed, but we would definitely be happy to accept a solution/improvement PR for this. 💯
